packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -479,6 +479,21 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 			.tox-tinymce {
 				border-top: none !important;
 			}
+			
+			/* Re-override Font Awesome styles that are otherwise overridden by TinyMCE: */
+			.plugin-icon.fa, .plugin-icon.far, .plugin-icon.fas {
+				font-family: "Font Awesome 5 Free";
+				font-size: 20px;
```

Comment:
The `font-size` is currently hardcoded. It may make sense to base this on `theme.toolbarHeight` instead.